onnx/defs/quantization/defs.cc

```diff
@@ -145,38 +145,57 @@ y = saturate (round (x / y_scale) + y_zero_point)
 
 ONNX_OPERATOR_SET_SCHEMA(
     DynamicQuantizeLinear,
-    11,
+    13,
     OpSchema()
-        .SetDoc(DynamicQuantizeLinear_ver11_doc)
+        .SetDoc(DynamicQuantizeLinear_ver13_doc)
         .Input(0, "x", "Input tensor", "T1")
         .Output(0, "y", "Quantized output tensor", "T2")
         .Output(1, "y_scale", "Output scale. It's a scalar, which means a per-tensor/layer quantization.", "tensor(float)")
         .Output(2, "y_zero_point", "Output zero point. It's a scalar, which means a per-tensor/layer quantization.", "T2")
+        .Attr(
+            "axis",
+            "(Optional) The axis of the quantization dimension of the input tensor. Negative value means counting dimensions from the back. Accepted range is [-r, r-1] where r = rank(input)",
+            AttributeProto::INT,
+            static_cast<int64_t>(1))
```

Comment:
There is an issue here. In QuantizeLinear, we use the size of scale to determine if it is per-tensor or per-channel, i.e., if size of scale is 1, then it is per-tensor; if size of scale is same the x.shape[axis], then it is a per-channel. Here we can not just use the axis to differentiate it.